WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -123,6 +125,51 @@ public void onClick(View v) {
         return view;
     }
 
+    public void handleEditPostResult(int resultCode, Intent data) {
+        if (resultCode == RESULT_OK && data != null && isAdded()) {
+
+            boolean hasChanges = data.getBooleanExtra(EditPostActivity.EXTRA_HAS_CHANGES, false);
+            final long postId = data.getLongExtra(EditPostActivity.EXTRA_POSTID, -1);
+            boolean isPublishable = data.getBooleanExtra(EditPostActivity.EXTRA_IS_PUBLISHABLE, false) && postId != -1;
+            boolean savedLocally = data.getBooleanExtra(EditPostActivity.EXTRA_SAVED_AS_LOCAL_DRAFT, false);
+
+            if (hasChanges) {
+                if (!NetworkUtils.isNetworkAvailable(getActivity())) {
+                    ToastUtils.showToast(getActivity(), R.string.error_publish_no_network, ToastUtils.Duration.SHORT);
+                } else {
+                    if (isPublishable) {
+                        if (savedLocally) {
+                            Snackbar.make(getActivity().findViewById(R.id.coordinator), R.string.editor_post_saved_locally_not_published, Snackbar.LENGTH_LONG)
```

Comment:
Let's wrap the long lines (this one and line 151) to 120char as per [the style guide](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/CODESTYLE.md). Thanks!